kornia/augmentation/augmentation.py

```diff
@@ -803,8 +872,18 @@ def apply_transform(
         transform = cast(torch.Tensor, transform)
         return affine(input, transform[..., :2, :3], self.resample.name.lower(), 'zeros', self.align_corners)
 
+    def inverse_transform(
+        self, input: torch.Tensor, transform: Optional[torch.Tensor] = None,
+        size: Optional[Tuple[int, int]] = None, **kwargs
+    ) -> torch.Tensor:
+        mode = self.resample.name.lower() if "mode" not in kwargs else kwargs['mode']
+        align_corners = self.align_corners if "align_corners" not in kwargs else kwargs['align_corners']
+        padding_mode = 'zeros' if "padding_mode" not in kwargs else kwargs['padding_mode']
+        transform = cast(torch.Tensor, transform)
+        return affine(input, transform.inverse()[..., :2, :3], mode, padding_mode, align_corners)
```

Comment:
remove boilerplate code and reuse in both places the affine function (or use directly warp_affine)